chain/client/src/test_utils/client.rs

```diff
@@ -61,31 +59,23 @@ impl Client {
         block: MaybeValidated<Block>,
         provenance: Provenance,
     ) -> Result<Vec<CryptoHash>, near_chain::Error> {
-        self.process_block_sync_with_produce_chunk_options(block, provenance, true, false)
+        self.process_block_sync_with_produce_chunk_options(block, provenance, false)
     }
 
-    pub fn process_block_test_no_produce_chunk(
+    pub fn process_block_test_allow_errors(
         &mut self,
         block: MaybeValidated<Block>,
         provenance: Provenance,
     ) -> Result<Vec<CryptoHash>, near_chain::Error> {
-        self.process_block_sync_with_produce_chunk_options(block, provenance, false, false)
-    }
-
-    pub fn process_block_test_no_produce_chunk_allow_errors(
```

Comment:
any tests that used this method for checking scenarios with missing chunks - quite an important and frequent case - will no longer do that